apps/constellation/constellation.hpp

```diff
@@ -139,18 +139,20 @@ class Constellation : public ledger::BlockSinkInterface
   using HttpModulePtr          = std::shared_ptr<HttpModule>;
   using HttpModules            = std::vector<HttpModulePtr>;
   using TransactionProcessor   = ledger::TransactionProcessor;
-  using TrustSystem            = p2p::P2PTrustBayRank<Muddle::Address>;
+  using TrustSystem            = p2p::P2PTrustBayRank<muddle::Address>;
```

Comment:
delete?

Reply:
Keep, we need to update in the interface and also do more testing on this subsystem